### PR TITLE
Cwc/use jwt language for endpoints industry

### PR DESCRIPTION
**Pull request by cwc-products**

### CODE REVIEW

File: app/controllers/_industry/_lib/_topOccs.options.js [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
-    await NAICSModel.find(query, projection)
-        .lean()
-        .exec()
-        .then(docs=>res.json(docs))
-        .catch(err=>res.status(500).send("unable to retrieve industry"))
-}
+module.exports = async function topOccsOptions(req, res) {
+  try {
+    // @todo the top occs have not been translated.
+    const { language = COMMON_CONSTANTS.LANGUAGE.ENGLISH } = 'en';// req.user;
```
Comment (cwc-products) [inline app/controllers/_industry/_lib/_topOccs.options.js]: @cfuhriman should I run the industry translations?

File: routes/_api/industry.js [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
-
-}
\ No newline at end of file
+module.exports = function industry(router, controller) {
+  // !Order counts!
+  router.route('/industry/top-occupations/options')
+    .get(controller.occs.topOptions)
+    .post(controller.occs.topOptions);
+
+  router.route('/industry/top-occupations')
```
Comment (cwc-products) [inline routes/_api/industry.js]: @cfuhriman The following two route did not appear in the swagger dock.  Do you remember what these routes were intended for?

File: app/controllers/_industry/_lib/_topOccs.js [hunk 3]
```diff
@@ -1,50 +1,52 @@
...
-    } else {
-        // need to cast NAICS, otherwise MONGO may try to process it as a number
-        query = Object.assign({}, query, {NAICS: NAICS.toString(), cipCode:cipCode});
-    }   
-    
-    if(institutionLevel) {
-        query = Object.assign({}, query, {institutionLevel:institutionLevel});
+module.exports = async function topOccs(req, res) {
+  try {
+    const { language = COMMON_CONSTANTS.LANGUAGE.ENGLISH } = req.user;
```
Comment (cwc-products) [inline app/controllers/_industry/_lib/_topOccs.js]: Confirm translation working after route added to swagger.

File: app/controllers/_industry/_lib/_topOccs.options.js (hunk 1 above)
Comment (cfuhriman) [reply app/controllers/_industry/_lib/_topOccs.options.js]: Not yet.   We need to create an identifier that isn't an ObjectID.

File: app/controllers/_industry/_lib/_topOccs.js (hunk 3 above)
Comment (cfuhriman) [reply app/controllers/_industry/_lib/_topOccs.js]: If they work then we should add to the swagger definitions

**Pull request merged**